api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/WorkspaceTargetProperty.java

```diff
@@ -14,7 +14,7 @@ public enum WorkspaceTargetProperty implements ModelBackedTargetProperty<Workspa
   TERRA_NAME("terra_name", Workspace::getTerraName),
   NAMESPACE("namespace", Workspace::getNamespace),
   CDR_VERSION_ID("cdr_version_id", Workspace::getCdrVersionId),
-  CREATOR("creator", Workspace::getCreator),
+  CREATOR("creator", workspace -> workspace.getCreator().getEmail()),
```

Comment:
please switch to `getUserName()` because email is deprecated.  We want to stop using "email" in most cases because it's ambiguous (contact email or username?) and we sometimes get it wrong.

I have a new draft to remove it (and also update UserRole): #8989

Reply:
Thanks for pointing this out